docs/reverse_proxy.md

```diff
@@ -119,8 +130,9 @@ frontend https
   # Matrix client traffic
   acl matrix-host hdr(host) -i matrix.example.com
   acl matrix-path path_beg /_matrix
+  acl synapse-client-path path_beg /_matrix
 
-  use_backend matrix if matrix-host matrix-path
+  use_backend matrix if matrix-host matrix-path || matrix-host synapse-client-path
```

Comment:
Could someone double-check this config is correct? I'm not too familiar with HAProxy.

Reply:
it's correct, but as above I'd suggest sticking with one ACL.